models/sync-media/README.md

```diff
@@ -0,0 +1,71 @@
+# Synchronized Media
+
+This is a WIP, based on current discussions between De Marque and EDRLab.
+
+## Synchronized Media Document
+
+[A separate document dedicated to the syntax](syntax.md) is available on this repo.
+
+## Listing documents
+
+### Collection-based
+
+*Note: Should we make the media type optional?*
+
+```json
+"sync-media": [
+  {
+    "href": "sync.json",
+    "type": "application/vnd.readium.sync-media+json"
+  },
+  {
+    "href": "sync2.json",
+    "type": "application/vnd.readium.sync-media+json"
+  }
+]
+```
+
+### Linked and chained
```

Comment:
This has the advantage of being more compact in the manifest, and allow to specify multiple sync media in a single publication (maybe with `rel` or additional metadata).

Reply:
I may not get the point, but I would expect a simple and natural way to define multiple synchronized reading orders.